test/fixtures/cts/1/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Let's also drop all the tests that aren't the `/0/` (which is the ts-node support)